### ISSUE TITLE
No proper display of GND URI

### ISSUE THREAD

**Issue by henning-gerhardt** [kitodo/kitodo-production#884]
After finding a solution for #882 it was now possible to store a new person but after new loading the meta data the displayed URI to this new stored record was wrong:

![gnd_error_002](https://user-images.githubusercontent.com/2129672/28413954-c9581876-6d49-11e7-8002-498c08be6137.png)

It was missing the identifier value 132775026.

Into the mets.xml the following was stored in Kitodo.Production 2.1.0

```xml
<goobi:metadata name="Author" type="person">
    <goobi:lastName>Forberger</goobi:lastName>
    <goobi:firstName>Rudolf</goobi:firstName>
    <goobi:authorityURI>http://d-nb.info/gnd/</goobi:authorityURI>
    <goobi:authorityValue>http://d-nb.info/gnd/132775026</goobi:authorityValue>
    <goobi:displayName>Forberger, Rudolf</goobi:displayName>
</goobi:metadata>
```

But if I tried look into the stored information with Kitodo.Production 2.0.0 I get the following

```xml
<goobi:metadata name="Author" type="person">
    <goobi:lastName>Forberger</goobi:lastName>
    <goobi:firstName>Rudolf</goobi:firstName>
    <goobi:authorityID>gnd</goobi:authorityID>
    <goobi:authorityURI>http://d-nb.info/gnd/</goobi:authorityURI>
    <goobi:authorityValue>http://d-nb.info/gnd/132775026</goobi:authorityValue>
    <goobi:displayName>Forberger, Rudolf</goobi:displayName>
</goobi:metadata>
```

with a proper displayed value for URI.

**Comment by henning-gerhardt** [comment]
If the missing (compare between 2.0.0 and 2.1.0)
```xml
<goobi:authorityID>gnd</goobi:authorityID>
```
is added into the `meta.xml` then is the displayed URI correct and stay correct after saving and reloading.

**Comment by henning-gerhardt** [comment]
Probably this issue is created by a configuration error and issue #883.

**Comment by henning-gerhardt** [comment]
Wrong storing and displaying of a GND entry was triggered by a configuration mistake. Loading GND information are only possible with fix of #883.
